tests/integration/test_task_pool.py

```diff
@@ -272,37 +268,31 @@ async def test_match_taskdefs(
     'items, expected_tasks_to_hold_ids, expected_warnings',
     [
         param(
-            ['foo.1', 'foo.2'], ['foo.1', 'foo.2'], [],
+            ['1/foo', '2/foo'], ['1/foo', '2/foo'], [],
             id="Active & future tasks"
         ),
         param(
-            ['*.1', '*.2'], ['foo.1', 'bar.1'],
-            ["No active tasks matching: *.2"],
+            ['1/*', '2/*'], ['1/foo', '1/bar'],
+            ["No active tasks matching: 2/*"],
             id="Name globs hold active tasks only"
         ),
         param(
-            ['FAM.1', 'FAM.2'], ['bar.1'],
-            ["No active tasks in the family 'FAM' matching: FAM.2"],
+            ['1/FAM', '2/FAM'], ['1/bar'],
+            ["No active tasks in the family 'FAM' matching: 2/FAM"],
             id="Family names hold active tasks only"
         ),
         param(
-            ['foo.*', 'bar', 'pub', 'grogu.*'], ['foo.1', 'bar.1'],
-            ["No active instances of task: pub",
-             "No active tasks matching: grogu.*"],
-            id="Point globs/point omitted hold active tasks only"
```

Comment:
Is it no longer possible to hold all instances of a task across cycle points?

Reply:
Can still hold all "active" instances of a task across all cycle points:

```console
$ cylc hold 'workflow//*/task'
```

Reply:
Then can the test be reimplemented as
```
        param(
            ['//*/foo', '*/bar', '*/pub', '//*/grogu'], ['1/foo', '1/bar'],
            ["No active tasks matching: */pub",
             "No active tasks matching: //*/grogu"],
            id="Point globs hold active tasks only"
        ),
```

Reply:
You can, tested in id_match.